scripts/exregional_run_vx_ensgrid.sh

```diff
@@ -106,35 +82,45 @@ NUM_PAD=${NDIGITS_ENSMEM_NAMES}
 #
 #-----------------------------------------------------------------------
 #
-# Create LOG_SUFFIX to read into METplus conf files.
+# Pick a directory structure for METplus output files
 #
 #-----------------------------------------------------------------------
 #
+if [ $RUN_ENVIR = "nco" ]; then
+    export INPUT_BASE=$COMIN
+    export OUTPUT_BASE=$COMOUT/metout
+    export MEM_BASE=$OUTPUT_BASE
+    export LOG_DIR=$LOGDIR
 
-if [ ${VAR} == "APCP" ]; then
-  LOG_SUFFIX=ensgrid_${CDATE}_${VAR}_${ACCUM}h
+    export POSTPRD=
+    export MEM_STAR=
+    export MEM_CUSTOM=
```

Comment:
It might be better to use `unset` here (and in other similar instances) so that if these are used later, it throws an error and we know we need more logic for NCO vs community mode.

Reply:
This is the hack I mentioned about in getting MET plus conf files to work with both NCO and COMMUNITY mode. It would need to be defined to an "empty string" since it is actually used the conf file, so I don't think `unset` will work. POSTPRD is not used in NCO mode but it is still there in the conf file.